src/daft-functions/src/numeric/round.rs

```diff
@@ -13,50 +13,38 @@ use serde::{Deserialize, Serialize};
 #[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Eq, Hash)]
 pub struct Round;
 
+#[derive(FunctionArgs)]
+struct RoundArgs<T> {
+    input: T,
+    #[arg(optional)]
+    decimals: Option<u32>,
```

Comment:
why do we need `arg(optional)` shouldn't we be able to infer that from the `Option` type?

Reply:
I did have that before but I chose to change it here because of two reasons:
1. it makes it more explicit so it's quick to see the args and determine which ones are optional
2. `Option<SomeLiteralType>` does also implement `FromLiteral`, and a function could theoretically take in a required nullable literal, which is semantically different from an optional literal. Just leaving the option (no pun intended) open for people.